bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/discovery/WemoLinkDiscoveryService.java

```diff
@@ -125,8 +125,13 @@ public void startScan() {
                     <u:GetEndDevices xmlns:u="urn:Belkin:service:bridge:1">\
                     <DevUDN>\
                     """
-                    + devUDN + "</DevUDN><ReqListType>PAIRED_LIST</ReqListType>" + "</u:GetEndDevices>" + "</s:Body>"
-                    + "</s:Envelope>";
+                    + devUDN + """
+                            </DevUDN>\
+                            <ReqListType>PAIRED_LIST</ReqListType>\
+                            </u:GetEndDevices>\
+                            </s:Body>\
+                            </s:Envelope>\
+                            """;
```

Comment:
Minor, but does the indentation have to be different in this section for any reason? I think it would be more readable with same indentation as the first section before the variable.

Reply:
Yes it looks ugly. But I could not convince spotless to accept it. I also tried moving the ```"""``` to the next line or ```+ """``` to the next line - but not success. Spotless always formatted it this way.

Reply:
Ah, okay. Let's leave it then. Spotless decided. 🙂